plugins/UsersManager/API.php

```diff
@@ -780,7 +819,23 @@ private function isUserTheOnlyUserHavingSuperUserAccess($userLogin)
     }
 
     /**
-     * Generates a unique MD5 for the given login & password
+     * Generates a new random authentication token.
+     *
+     * @param string $userLogin Login
+     * @return string
+     */
+    public function createTokenAuth($userLogin)
+    {
+        return hash(
```

Comment:
It was already changed by @tsteur, but the original intention was to change the length of the hashes to be able to keep the "changing password changes token" in place. It might have been sha1 as well, there was no real argument it had to be sha256.

If I had known the length checks in the SDKs I would have probably chosen something different :+1: